streaming/index.js

```diff
@@ -111,6 +111,35 @@ const startServer = async () => {
   const server = http.createServer();
   const wss = new WebSocketServer({ noServer: true });
 
+  /**
+   * Adds a namespace to Redis keys or channel names
+   * Fixes: https://github.com/redis/ioredis/issues/1910
+   * @param {string} keyOrChannel
+   * @returns {string}
+   */
+  function redisNamespaced(keyOrChannel) {
+    if (redisConfig.namespace) {
+      return `${redisConfig.namespace}:${keyOrChannel}`;
+    } else {
+      return keyOrChannel;
+    }
+  }
+
+  /**
+   * Removes the redis namespace from a channel name
+   * @param {string} channel
+   * @returns {string}
+   */
+  function redisUnnamespaced(channel) {
+    if (typeof redisConfig.namespace === "string") {
+      // Note: this removes the configured namespace and the colon that is used
+      // to separate it:
+      return channel.slice(redisConfig.namespace.length + 1);
+    } else {
+      return channel;
+    }
+  }
```

Comment:
I don't really like these "floating" functions, because it makes code very hard to split up later, ideally we'd have something identical to `Redis::Namespaced` from the ruby side, but for JavaScript that would wrap a Redis Client, I think. Though I'm not really wanting to get into the whole "wrapping redis clients" thing right now.